en/airframes/airframe_reference.md

```diff
@@ -1052,3 +1052,21 @@ This page lists all supported airframes and types including
 </tr>
 </tbody></table>
 
+
```

Comment:
Please delete this file/change - this file is autogenerated from the work you did in Firmware repo. It is already there: https://dev.px4.io/master/en/airframes/airframe_reference.html#vtol-octoplane

But you do need to add the image :-)

Reply:
Oh, sure :)